soft65c02_tester/src/executor.rs

```diff
@@ -6,12 +6,13 @@ use std::{
 use anyhow::anyhow;
 use soft65c02_lib::{Memory, Registers};
 
-use crate::{AppResult, CliCommand, CliCommandParser, Command, OutputToken};
+use crate::{AppResult, CliCommand, CliCommandParser, Command, OutputToken, symbols::SymbolTable};
```

Comment:
Maybe we should see how to expose libs in a different consistent way in `lib.rs` (and I like your approach). For now, is it possible to just expose `SymbolTable` as the other structures?

Reply:
Ah yes, you mean expose it with a "pub use symbols::SymbolTable;" in lib.rs, and just use "SymbolTable" in executor.rs.

I've got another PR lined up after this one (I've been extremely busy over the weekend and bank holiday in the UK just doing updates to this repo).
I've add the fix there for the change to the module usage, rather than change this one and go through the merge fun on my local repo if you don't mind.